Sources/SwiftParser/Availability.swift

```diff
@@ -95,7 +95,7 @@ extension Parser {
     }
   }
 
-  mutating func parseAvailabilityArgumentSpecList() -> RawAvailabilitySpecListSyntax {
+  mutating func parseAvailabilityArguments() -> RawAvailabilityArgumentListSyntax {
```

Comment:
The functions `parseAvailabilitySpecList` and `parseAvailabilityArgumentSpecList` perform similar operations, but there are subtle differences. I tried to rename them to something other than `Spec`, but I couldn't think of a suitable name. Is there a better approach? 🤔